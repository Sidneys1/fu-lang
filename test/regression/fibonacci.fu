fib: u32(n: u32) = {
  if (n < 2) return n;
  return fib(n - 1) + fib(n - 2);
};

main: u32() => fib(12); // 144
