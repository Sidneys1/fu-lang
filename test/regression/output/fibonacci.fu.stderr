INFO:fu.compiler:Output binary `a.out`: 638 Bytes
